src/main/java/io/jenkins/plugins/appcenter/AppCenterRecorder.java

```diff
@@ -141,6 +146,7 @@ public FormValidation doCheckOwnerName(@QueryParameter String value) {
                 return FormValidation.error(Messages.AppCenterRecorder_DescriptorImpl_errors_missingOwnerName());
             }
 
+            // TODO proper validator, every non-empty string should be valid?
```

Comment:
If you try and change your username in the AppCenter UI on their website and enter, for example, a `:` character then the error tells you that:

>Username can only contain letters, numbers, dashes, underscores, or periods.

So I would remove this TODO.

Reply:
Yeah, that should be removed. It's a left-over from before I put in the `DistributionGroupValidator`.